features/support/helpers.rb

```diff
@@ -110,26 +114,27 @@ def run_in_shell(*args)
 
   p.value
 end
+# rubocop:enable Metrics/AbcSize
 
 #
 
 def slug(title = nil)
   if !title
     then Time.now.strftime("%s%9N") # nanoseconds since the Epoch
-    else title.downcase.gsub(/[^\w]/, " ").strip.gsub(/\s+/, '-')
+  else title.downcase.gsub(/[^\w]/, " ").strip.gsub(/\s+/, "-")
   end
 end
 
 #
```

Comment:
I audit code so much in my daily life that normally add comments between methods and where there are no comments I either leave "# --" or "#" because it helps to spot a coming method quickly (lets admit it, we all have trouble finding a def at some point when looking through a large file or object.)

@parkr doesn't like it so when he mentions it I remove them (as my editor -- Atom -- has a plugin I built to automatically do that.) Some may argue that's why we have symbol finders, but I argue that when auditing code, I'm hardly interested in that and at a normal basis I am too because I feel that a file that cannot be browsed without a symbol browser and without syntax highlighting is no code that should be unleashed upon the world.

Long explanation, sorry.